pkg/anonymiser/anonymiser.go

```diff
@@ -25,26 +33,25 @@ func NewAnonymiser(source reader.Reader, tables config.Tables) reader.Reader {
 	return &anonymiser{source, tables}
 }
 
-func (a *anonymiser) ReadTable(tableName string, rowChan chan<- database.Row, opts reader.ReadTableOpt) error {
+// ReadTable wraps reader.ReadTable method for anonymising rows published from the reader.Reader
+func (a *anonymiser) ReadTable(tableName string, rowChan chan<- database.Row, opts reader.ReadTableOpt, configTables config.Tables) error {
 	logger := log.WithField("table", tableName)
-
 	logger.Debug("Loading anonymiser config")
 	table, err := a.tables.FindByName(tableName)
 	if err != nil {
 		logger.WithError(err).Debug("the table is not configured to be anonymised")
-		return a.Reader.ReadTable(tableName, rowChan, opts)
+		return a.Reader.ReadTable(tableName, rowChan, opts, configTables)
```

Comment:
Maybe we should start thinking about a context?

Reply:
This came to my my mind for managing query timeouts, however I still didn't find what would be a proper timeout value for reading and writing, that's why I didn't implemented. Is that what you mean?

Reply:
@boekkooi-fresh let's add this as a next task so I can move my ticket ^^ wdyt?